src/main/java/com/aspectsecurity/contrast/contrastjenkins/VulnerabilityTrendStep.java

```diff
@@ -178,6 +179,9 @@ public Step newInstance(Map<String, Object> arguments) {
             if (arguments.containsKey("queryBy")) {
                 Object queryBy = arguments.get("queryBy");
 
+                step.setQueryBy((int) queryBy);
+            } else if (arguments.containsKey("appVersionTagFormat")) {
```

Comment:
Why did another option get added to the pipeline?

Reply:
We had this option in the previous plugin version, so I added it for better compatibility.